operations/restrictions.md

```diff
@@ -471,7 +473,106 @@ Note the `StartUtc` and `EndUtc` properties must be set to the midnight of the g
 | `MinPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the minimum price per time unit. |
 | `MaxPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the maximum price per time unit. |
 
+## Clear restrictions
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change. Use of this operation must be enabled per enterprise. Please contact the Technical Partner Support team in order to enable it.
+
+Deletes restrictions that [match the conditions](#matching-conditions) using the [splicing algorithm](#splicing-algorithm).
+
+This operation is meant to replace the [restrictions delete](#delete-restrictions) operation as the introduction of [restrictions set](#set-restrictions) makes the delete operation very impractical. This is because the [merging algorithm](#merging-algorithm) makes it very hard to keep track of individual restriction ids.
```

Comment:
I would be inclined to reduce all this right down and simply say "This operation can be used in conjunction with [Set restrictions](#set-restrictions)." or "This operation is intended to be used in conjunction with [Set restrictions](#set-restrictions)."